scripts/smart-dump.ts

```diff
@@ -29,7 +30,7 @@ const exec = cmd => {
   }
 };
 
-program.command('dump [recipe] [env] [as_user]').action(async (recipe, env, asUser) => {
+program.command('dump [recipe] [as_user]').action(async (recipe, asUser, env) => {
```

Comment:
Why not include `[env]` in the command description since we're using it?

Reply:
Because this is part of the `npm run script` preloader. In reality, we're just getting env vars in this script.